jwt.js

```diff
@@ -48,6 +53,7 @@ function fastifyJwt (fastify, options, next) {
   const decodeOptions = options.decode || {}
   const signOptions = options.sign || {}
   const verifyOptions = options.verify || {}
+  const messagesOptions = Object.assign(messages, options.messages)
```

Comment:
We should write {}, messages, option in order to don't change the messages object

Reply:
Thats true, `Object.assign()` will modify the source i.e. the messages object. Let me see if I can avoid this.

Reply:
Exactly, sorry if I wrote like it, but I was travelling 😅